tests/test_MPMD/test_1/main.py

```diff
@@ -8,15 +8,9 @@
 # License: BSD-3-Clause-LBNL
 # Authors: Bhargav Sriram Siddani, Revathi Jambunathan, Edoardo Zoni, Olga Shapoval, David Grote, Axel Huebl
 
-import mpi4py
```

Comment:
The following lines:
```
import mpi4py
mpi4py.rc.initialize = False
mpi4py.rc.finalize = False
```
were resulting in `MPI initialized with thread support level 0` instead of `MPI initialized with thread support level 3`.

Removing these lines results in `MPI initialized with thread support level 3` and avoids the following error:
```
Abort(3755781) on node 10 (rank 10 in comm 0): Fatal error in internal_Comm_free: Invalid communicator, error stack:
internal_Comm_free(87): MPI_Comm_free(comm=0x7f03e8f576b0) failed
internal_Comm_free(44): Null communicator
```
Need to look into this further.

Reply:
Slack: solved :tada: